tests/config/pytest_modules.yml

```diff
@@ -2227,9 +2227,6 @@ subworkflows/fastq_align_chromap:
 subworkflows/fastq_align_dna:
   - subworkflows/nf-core/fastq_align_dna/**
   - tests/subworkflows/nf-core/fastq_align_dna/**
-subworkflows/fastq_align_star:
-  - subworkflows/nf-core/fastq_align_star/**
-  - tests/subworkflows/nf-core/fastq_align_star/**
```

Comment:
?

Reply:
I got a linting error for that subworkflow because there's an nf-test  but it wasn't removed from the pytests